presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -231,6 +233,45 @@ public void testHiveViewInInformationSchema()
                 .contains(row("n_nationkey", "bigint", "", ""));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testFailingHiveViewsForInformationSchema()
+    {
+        if (getHiveVersionMajor() < 3) {
+            // Skip older versions since they don't support "getTableNamesByType" in ThriftHiveMetastore#getAllViews
+            throw new SkipException("Only test the behavior for hive 3");
```

Comment:
Maybe add a bit more explanation:
```
// Skip older versions since they don't support "getTableNamesByType" in ThriftHiveMetastore#getAllViews
// and so information_schema.views does not include translated Hive views properly
```

BTW, it still worthwhile to run the test with older Hive versions, just the assertions would be different -- to document current behavior & prevent regressions. 
How would it look like if you used `getHiveVersionMajor()` on per-assertion basis?
kind of like https://github.com/prestosql/presto/blob/3c5b5b053affa6cf093e7f4cbca0bb26d9bfb380/presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java#L129-L131

Reply:
Can you also add a test to `TestHiveViewsLegacy.java` to document behavior difference.